### PR TITLE
Migrate config/weback over to mjs

### PR DESCRIPTION
**Pull request by DanDroryAu**

### CODE REVIEW

File: packages/sku/config/webpack/plugins/metrics-plugin/index.mjs [hunk 1]
```diff
@@ -1,7 +1,9 @@
-const { performance } = require('node:perf_hooks');
-const prettyMilliseconds = require('pretty-ms');
-const debug = require('debug')('sku:metrics');
-const track = require('../../../../telemetry');
+import { performance } from 'node:perf_hooks';
+import prettyMilliseconds from 'pretty-ms';
+import debug from 'debug';
+import { timing } from '../../../../telemetry';
+
+debug('sku:metrics');
```
Comment (askoufis) [inline packages/sku/config/webpack/plugins/metrics-plugin/index.mjs:6]: Should this be assigned to something?

**Review by askoufis — APPROVED** [review]

File: packages/sku/config/webpack/plugins/metrics-plugin/index.mjs (hunk 1 above)
Comment (DanDroryAu) [reply packages/sku/config/webpack/plugins/metrics-plugin/index.mjs:6]: It should be a `const log = debug('sku:metrics');`. I'll update.

Comment (DanDroryAu) [reply packages/sku/config/webpack/plugins/metrics-plugin/index.mjs:6]: Also renaming the rest of the debug calls to log.

Comment (DanDroryAu) [reply packages/sku/config/webpack/plugins/metrics-plugin/index.mjs]: I've found some other instances of this mistake. I've added them to this pr.

**Pull request merged**